spec/notification-manager-spec.js

```diff
@@ -66,4 +66,27 @@ describe('NotificationManager', () => {
       expect(notification.getType()).toBe('success')
     })
   })
+
+  describe('clearing notifications', function () {
+    it('clears the notifications when ::clear has been called', function(){
+      manager.addSuccess('success')
+      expect(manager.getNotifications().length).toBe(1)
+      manager.clear()
+      expect(manager.getNotifications().length).toBe(0)
+    })
+
+    describe('adding events', () => {
+      let addSpy
```

Comment:
The name `addSpy` doesn't make sense in this context. It looks like that name is used in the previous `describe` because it is an event listener that is registered using `onDidAddNotification`. I think `clearSpy` would make more sense here.